code/index.html

```diff
@@ -1,22 +1,118 @@
+      <br> This week on ARCHMAG, insurance company Hadid`s of London revealed it would overhaul the iconic Underwriting
+      Room at the heart of its high-tech building in London designed by Henry Rogers.
+      <br>
+      <a href="#"> <i> Read more>> </i></a>
+    </h2>
+  </section>
+  <hr>
+
+  <a class="top-news-text" href="#">LATEST NEWS</a>
+
+  <!-- The other news  -->
+
+  <section class="news-cards">
+
+    <div class="card">
+      <h4>DESIGN</h4>
+      <img class="news-image" src="./images/lance.jpg" alt="Lance">
+      <p>The architect changed the original house by making the garden participate actively in the composition of the house concept. <br>
+        <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>AWARD</h4>
+      <img class="news-image" src="./images/chris.jpg" alt="Chris">
+      <p>For coffered ceiling inside the entrance atrium architect first used in domestic architecture, glass block ceiling. <br> <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>INTERIOR</h4>
+      <img class="news-image" src="./images/martin.jpg" alt="Martin">
+      <p>On the street facade highlights the many tall windows with white woodwork and shutters in one sector and the other cut on the stone. <br><a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>INTERIOR</h4>
+      <img class="news-image" src="./images/knubb.jpg" alt="knubb">
+      <p>These two contrasting spaces that complement each other perfectly are created by an arched volume of wood. <br>
+        <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>NEWS</h4>
+      <img class="news-image" src="./images/pawel.jpg" alt="Pawel">
+      <p>The long, narrow site designated for the new library presented some technical challenges, including the need to consider the future construction of a tunnel. <br> <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>AWARDS</h4>
+      <img class="news-image" src="./images/pierre.jpg" alt="Pierre">
+      <p>An important part of the concept of this house was not only the treatment of light and water, but also the sound of the wind. <br> <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>DESIGN</h4>
+      <img class="news-image" src="./images/adrien.jpg" alt="Adrien">
+      <p>The building is a public space owned by citizens and open to all people. Its design was done by encouraging active participation among residents and partners. <br> <a href="#">Read more >></a> </p>
+    </div>
+    <div class="card">
+      <h4>OFFICE</h4>
+      <img class="news-image" src="./images/patrick.jpg" alt="Patrick">
+      <p>The combination of its steel and glass structure with the wood of the facade results in a striking building.
+        <br><a href="#">Read more >></a> </p>
```

Comment:
I think that the guidelines told us not to use `<br>`?

Reply:
I taught it would be ok if it is used in texts, but probably there should be sth more modern than <br> tag. I will check for it.